Client/src/app/components/jokes/joke-detail/joke-detail.component.ts

```diff
@@ -5,7 +5,7 @@ import { Joke } from '../../../_models/joke';
 import { JokesService } from 'src/app/_services/jokes.service';
 
 @Component({
-  selector: '',
+  selector: 'add-joke-detail',
```

Comment:
joke-detail is enough

Reply:
It was already there. But now I decided to delete it